### ISSUE TITLE
Is it expected? "System.err: java.lang.NullPointerException: Attempt to invoke virtual method 'int java.lang.String.length()'"

### ISSUE THREAD

**Issue by pis0** [myflashlab/common-dependencies-ANE#30]
android logcat stacks related to overrideAir v5.6.1:

getting the same error using SDKs 31.0.0.96 (release) / 32.0.0.81 (off version)

```
11-27 16:22:51.234 32358 32358 W System.err: java.lang.NullPointerException: Attempt to invoke virtual method 'int java.lang.String.length()' on a null object reference
11-27 16:22:51.238 32358 32358 W System.err:    at com.myflashlab.dependency.overrideAir.AirCommand.decode(AirCommand.java:93)
11-27 16:22:51.238 32358 32358 W System.err:    at com.myflashlab.dependency.overrideAir.AirCommand.call(AirCommand.java:58)
11-27 16:22:51.238 32358 32358 W System.err:    at com.adobe.air.Entrypoints.EntryMainWrapper(Native Method)
11-27 16:22:51.238 32358 32358 W System.err:    at com.adobe.air.Entrypoints.EntryMain(Entrypoints.java:139)
11-27 16:22:51.238 32358 32358 W System.err:    at com.adobe.air.AndroidActivityWrapper.LaunchApplication(AndroidActivityWrapper.java:1167)
11-27 16:22:51.238 32358 32358 W System.err:    at com.adobe.air.AndroidActivityWrapper.onSurfaceInitialized(AndroidActivityWrapper.java:1416)
11-27 16:22:51.238 32358 32358 W System.err:    at com.adobe.air.AIRWindowSurfaceView.surfaceChanged(AIRWindowSurfaceView.java:790)
11-27 16:22:51.238 32358 32358 W System.err:    at android.view.SurfaceView.updateWindow(SurfaceView.java:594)
11-27 16:22:51.238 32358 32358 W System.err:    at android.view.SurfaceView$3.onPreDraw(SurfaceView.java:177)
11-27 16:22:51.238 32358 32358 W System.err:    at android.view.ViewTreeObserver.dispatchOnPreDraw(ViewTreeObserver.java:944)
11-27 16:22:51.238 32358 32358 W System.err:    at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2055)
11-27 16:22:51.238 32358 32358 W System.err:    at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1107)
11-27 16:22:51.238 32358 32358 W System.err:    at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:6013)
11-27 16:22:51.238 32358 32358 W System.err:    at android.view.Choreographer$CallbackRecord.run(Choreographer.java:858)
11-27 16:22:51.239 32358 32358 W System.err:    at android.view.Choreographer.doCallbacks(Choreographer.java:670)
11-27 16:22:51.239 32358 32358 W System.err:    at android.view.Choreographer.doFrame(Choreographer.java:606)
11-27 16:22:51.239 32358 32358 W System.err:    at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:844)
11-27 16:22:51.239 32358 32358 W System.err:    at android.os.Handler.handleCallback(Handler.java:739)
11-27 16:22:51.239 32358 32358 W System.err:    at android.os.Handler.dispatchMessage(Handler.java:95)
11-27 16:22:51.239 32358 32358 W System.err:    at android.os.Looper.loop(Looper.java:148)
11-27 16:22:51.239 32358 32358 W System.err:    at android.app.ActivityThread.main(ActivityThread.java:5417)
11-27 16:22:51.239 32358 32358 W System.err:    at java.lang.reflect.Method.invoke(Native Method)
11-27 16:22:51.239 32358 32358 W System.err:    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
11-27 16:22:51.239 32358 32358 W System.err:    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
11-27 16:22:51.239 32358 32358 I ooo     : End of input at character 0 of
11-27 16:22:51.239 32358 32358 W System.err: org.json.JSONException: End of input at character 0 of
11-27 16:22:51.240 32358 32358 W System.err:    at org.json.JSONTokener.syntaxError(JSONTokener.java:449)
11-27 16:22:51.240 32358 32358 W System.err:    at org.json.JSONTokener.nextValue(JSONTokener.java:97)
11-27 16:22:51.240 32358 32358 W System.err:    at org.json.JSONObject.<init>(JSONObject.java:156)
11-27 16:22:51.240 32358 32358 W System.err:    at org.json.JSONObject.<init>(JSONObject.java:173)
11-27 16:22:51.240 32358 32358 W System.err:    at com.myflashlab.dependency.overrideAir.AirCommand.call(AirCommand.java:58)
11-27 16:22:51.240 32358 32358 W System.err:    at com.adobe.air.Entrypoints.EntryMainWrapper(Native Method)
11-27 16:22:51.240 32358 32358 W System.err:    at com.adobe.air.Entrypoints.EntryMain(Entrypoints.java:139)
11-27 16:22:51.240 32358 32358 W System.err:    at com.adobe.air.AndroidActivityWrapper.LaunchApplication(AndroidActivityWrapper.java:1167)
11-27 16:22:51.240 32358 32358 W System.err:    at com.adobe.air.AndroidActivityWrapper.onSurfaceInitialized(AndroidActivityWrapper.java:1416)
11-27 16:22:51.240 32358 32358 W System.err:    at com.adobe.air.AIRWindowSurfaceView.surfaceChanged(AIRWindowSurfaceView.java:790)
11-27 16:22:51.240 32358 32358 W System.err:    at android.view.SurfaceView.updateWindow(SurfaceView.java:594)
11-27 16:22:51.240 32358 32358 W System.err:    at android.view.SurfaceView$3.onPreDraw(SurfaceView.java:177)
11-27 16:22:51.240 32358 32358 W System.err:    at android.view.ViewTreeObserver.dispatchOnPreDraw(ViewTreeObserver.java:944)
11-27 16:22:51.240 32358 32358 W System.err:    at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2055)
11-27 16:22:51.240 32358 32358 W System.err:    at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1107)
11-27 16:22:51.240 32358 32358 W System.err:    at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:6013)
11-27 16:22:51.240 32358 32358 W System.err:    at android.view.Choreographer$CallbackRecord.run(Choreographer.java:858)
11-27 16:22:51.240 32358 32358 W System.err:    at android.view.Choreographer.doCallbacks(Choreographer.java:670)
11-27 16:22:51.241 32358 32358 W System.err:    at android.view.Choreographer.doFrame(Choreographer.java:606)
11-27 16:22:51.241 32358 32358 W System.err:    at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:844)
11-27 16:22:51.241 32358 32358 W System.err:    at android.os.Handler.handleCallback(Handler.java:739)
11-27 16:22:51.241 32358 32358 W System.err:    at android.os.Handler.dispatchMessage(Handler.java:95)
11-27 16:22:51.241 32358 32358 W System.err:    at android.os.Looper.loop(Looper.java:148)
11-27 16:22:51.241 32358 32358 W System.err:    at android.app.ActivityThread.main(ActivityThread.java:5417)
11-27 16:22:51.241 32358 32358 W System.err:    at java.lang.reflect.Method.invoke(Native Method)
11-27 16:22:51.241 32358 32358 W System.err:    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
11-27 16:22:51.241 32358 32358 W System.err:    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
```

_Originally posted by @pis0 in https://github.com/myflashlab/common-dependencies-ANE/issues/27#issuecomment-441768694_

**Comment by myflashlab** [comment]
Thanks for reporting this. Although that's a soft warning in logcat and could safely ignored, we fixed that now in [V5.6.2](https://github.com/myflashlab/common-dependencies-ANE/tree/master/overridAir)

**Comment by pis0** [comment]
tks!
:)
